### PR TITLE
[Ameba]Update scripts/build

### PR DESCRIPTION
**Pull request by pankore**
#### Problem
* Update scripts/build files for Ameba platform

#### Change overview
* Add ameba-amebad-light in scripts/build files

#### Testing
build_example build

### CODE REVIEW

**Review by andy31415 — APPROVED** [review]

**Comment by andy31415** [comment]
fast track: small PR, build changes only

File: scripts/build/builders/ameba.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 
 class AmebaBoard(Enum):
     AMEBAD = auto()
+    LIGHT = auto()
```
Comment (andy31415) [inline scripts/build/builders/ameba.py:24]: This should probably be in AmebaApp not board.

CI fails because of this.

Comment (pankore) [reply scripts/build/builders/ameba.py]: Oh right, that was a typo, thanks for pointing it out!

**Review by woody-apple — APPROVED** [review]

**Pull request merged**